pkg/distributor/push.go

```diff
@@ -303,19 +302,19 @@ func calculateRetryAfter(retryAttemptHeader string, baseSeconds int, maxBackoffE
 	return strconv.FormatInt(delaySeconds, 10)
 }
 
-// toGRPCHTTPStatus converts the given error into an appropriate GRPC and HTTP status corresponding
+// toHTTPStatus converts the given error into an appropriate GRPC and HTTP status corresponding
```

Comment:
```suggestion
// toHTTPStatus converts the given error into an appropriate HTTP status corresponding
// to that error, if the error is one of the errors from this package. Otherwise, an
// http.StatusInternalServerError is returned.
```